build error: gcc12 -Werror=format-security

```
/build/darknet-alexeyab-git/src/darknet/src/detector.c: In function 'print_cocos':
/build/darknet-alexeyab-git/src/darknet/src/detector.c:495:29: error: format not a string literal and no format arguments [-Werror=format-security]
  495 |                 fprintf(fp, buff);
      |                             ^~~~
compilation terminated due to -Wfatal-errors.
```

compiler: gcc 12.1.0

--- a/src/detector.c
+++ b/src/detector.c
@@ -492,7 +492,7 @@ static void print_cocos(FILE *fp, char *image_path, detection *dets, int num_box
             if (dets[i].prob[j] > 0) {
                 char buff[1024];
                 sprintf(buff, "{\"image_id\":%d, \"category_id\":%d, \"bbox\":[%f, %f, %f, %f], \"score\":%f},\n", image_id, coco_ids[j], bx, by, bw, bh, dets[i].prob[j]);
-                fprintf(fp, buff);
+                fprintf(fp, "%s", buff);
                 //printf("%s", buff);
             }
         }

